### ISSUE TITLE
ValueError: Unsupported Ops in the model before optimization `CropAndResize` [Converter]

### ISSUE THREAD

**Issue by kerschung** [tensorflow/tfjs#736]
I want convert some existing model to web model format like [faster_rcnn_inception_v2_coco](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md). I check the ops doc is already supported `CropAndResize`, but i still facing the error.

I am new in tensorflow. Can someone tell me where is going wrong?
Thank you!!

TensorFlow.js: 0.6.0

```
tensorflowjs_converter \
    --input_format=tf_frozen_model \
    --output_node_names='detection_boxes,detection_scores,detection_classes,num_detections' \
    --saved_model_tags=serve \
    ./faster_rcnn_inception_v2_coco_2018_01_28/frozen_inference_graph.pb \
    ./faster_rcnn_inception_v2/web_model
```

When i use tfjs converter to convert exist model, the error shows as the following:
```
Using TensorFlow backend.
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/envs/tensorflow_p36/bin/tensorflowjs_converter", line 11, in <module>
    sys.exit(main())
  File "/home/ubuntu/anaconda3/envs/tensorflow_p36/lib/python3.6/site-packages/tensorflowjs/converters/converter.py", line 272, in main
    strip_debug_ops=FLAGS.strip_debug_ops)
  File "/home/ubuntu/anaconda3/envs/tensorflow_p36/lib/python3.6/site-packages/tensorflowjs/converters/tf_saved_model_conversion.py", line 325, in convert_tf_frozen_model
    skip_op_check=skip_op_check, strip_debug_ops=strip_debug_ops)
  File "/home/ubuntu/anaconda3/envs/tensorflow_p36/lib/python3.6/site-packages/tensorflowjs/converters/tf_saved_model_conversion.py", line 113, in optimize_graph
    ', '.join(unsupported))
ValueError: Unsupported Ops in the model before optimization
`CropAndResize`
```

**Comment by pyu10055** [comment]
@kerschung Please try again using the newly released tensorflowjs pip package (0.6.1). thanks.

**Comment by kerschung** [comment]
@pyu10055 It works! Thank you! I close the issue.
